.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Usage question
+    url: https://neurostars.org/tag/xcp_d
```

Comment:
One issue is that this tag doesn't exist yet. We'll need someone to create an issue on NeuroStars with the new tag for the page to exist.